endpoints/api/organization.py

```diff
@@ -372,16 +366,27 @@ def get(self, orgname):
             organization = model.organization.get_organization(orgname)
             private_repos = model.user.get_private_repo_count(organization.username)
             data = {"privateAllowed": False}
+            repos_allowed = 0
 
             if organization.stripe_id:
                 cus = stripe.Customer.retrieve(organization.stripe_id)
                 if cus.subscription:
-                    repos_allowed = 0
                     plan = get_plan(cus.subscription.plan.id)
                     if plan:
                         repos_allowed = plan["privateRepos"]
 
-                    data["privateAllowed"] = private_repos < repos_allowed
+            if features.RH_MARKETPLACE:
+                query = organization_skus.get_org_subscriptions(organization.id)
+                rh_subscriptions = list(query.dicts()) if query is not None else []
+                for subscription in rh_subscriptions:
+                    subscription_sku = marketplace_subscriptions.get_subscription_sku(
+                        subscription["subscription_id"]
+                    )
+                    equivalent_stripe_plan = get_plan_using_rh_sku(subscription_sku)
+                    if equivalent_stripe_plan:
+                        repos_allowed += equivalent_stripe_plan["privateRepos"]
+
+            data["privateAllowed"] = private_repos < repos_allowed
```

Comment:
Do we only return the number of private repos allowed as part of the response, or is the list of bound SKUs also included?

Reply:
I think this actually just returns a boolean value for whether you're allowed to create a private repo and the number of existing private repos